requirements_all.txt

```diff
@@ -245,7 +245,7 @@ alpha_vantage==2.2.0
 ambiclimate==0.2.1
 
 # homeassistant.components.amcrest
-amcrest==1.7.0
+amcrest==1.7.2
```

Comment:
AFAICT this release does not exist

Reply:
Resolved above: the release is on PyPI even if it is not on GitHub.

Reply:
now also on github